oauth2_http/java/com/google/auth/oauth2/ServiceAccountJwtAccessCredentials.java

```diff
@@ -203,7 +209,32 @@ public static ServiceAccountJwtAccessCredentials fromPkcs8(
       throws IOException {
     PrivateKey privateKey = ServiceAccountCredentials.privateKeyFromPkcs8(privateKeyPkcs8);
     return new ServiceAccountJwtAccessCredentials(
-        clientId, clientEmail, privateKey, privateKeyId, defaultAudience);
+        clientId, clientEmail, privateKey, privateKeyId, defaultAudience, null);
+  }
+
+  /**
+   * Factory using PKCS#8 for the private key.
+   *
+   * @param clientId Client ID of the service account from the console. May be null.
+   * @param clientEmail Client email address of the service account from the console.
+   * @param privateKeyPkcs8 RSA private key object for the service account in PKCS#8 format.
+   * @param privateKeyId Private key identifier for the service account. May be null.
+   * @param defaultAudience Audience to use if not provided by transport. May be null.
+   * @param quotaProjectId The project used for quota and billing purposes. May be null.
+   * @return New ServiceAccountJwtAcceessCredentials created from a private key.
+   * @throws IOException if the credential cannot be created from the private key.
+   */
+  public static ServiceAccountJwtAccessCredentials fromPkcs8(
```

Comment:
This one is also unnecessary

Reply:
Is this one actually needed?

Reply:
I would say yes actually for this one. I moved it to package scope though as to not expose more surface to the api.